scripts/usb_scan.py

```diff
@@ -435,20 +433,19 @@ def parse_line(self, line):
         # 4. parse key=value pairs
         # pattern = r"\s*(class|subclass|prot|vid|pid|rel)\s*=\s*([0-9a-f]+)"
         last_end = 0
-        for matchNum, match in enumerate(re.finditer(self._PATTERN, target,
-                                                     re.IGNORECASE)):
-            if last_end != match.start():
-                self.parse_error(last_end, match.start(), target, line)
+        for _match in re.finditer(self._PATTERN, target, re.IGNORECASE):
```

Comment:
I like this removal of the not needed enumerate(), but what is the coding rule for renaming match to _match here?

I did not see other examples for single pre underscored variables in this code base here, so it looks inconsistent. Furthermore, I also found no explanation that would match its use in a local loop variable on a quick search.

Reply:
The change here is the avoidance ofusing the same name as a builtin variable,  `match` is a function in builtin module `re.match`.